packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsCalendarChannelsContainer.tsx

```diff
@@ -52,13 +52,16 @@ export const SettingsAccountsCalendarChannelsContainer = () => {
   if (!calendarChannels.length) {
     return <SettingsAccountsListEmptyStateCard />;
   }
+  const hasMultipleAccounts = accounts.length > 1;
 
   return (
     <>
-      <TabList
-        tabListId={SETTINGS_ACCOUNT_CALENDAR_CHANNELS_TAB_LIST_COMPONENT_ID}
-        tabs={tabs}
-      />
+      <div style={{ display: hasMultipleAccounts ? 'block' : 'none' }}>
```

Comment:
@ViditJain123 In React we should do something like {hasMultipleAccounts && <TabList ... />}

Reply:
> @ViditJain123 In React we should do something like {hasMultipleAccounts && <TabList ... />}

I tried doing that but for some reason, its hiding every component on the screen when handled that way..